docs/requirements-docs.txt

```diff
@@ -1,9 +1,5 @@
 numpydoc
 sphinx
 sphinx_rtd_theme
-tornado
-toolz
-zict
```

Comment:
The removed packages are dependencies of `streamz`. If all works well, they will be installed by the package install now defined within the RTD configuration.